doc/components.md

```diff
@@ -56,28 +55,19 @@ This will create a simple three frame animation
 
 ```dart
     List<Sprite> sprites = [0, 1, 2].map((i) => new Sprite('player_${i}.png')).toList();
-    this.player = AnimationComponent(64.0, 64.0, new Animation.spriteList(sprites, stepTime: 0.01));
+    final size = Vector2(64.0, 64.0);
+    this.player = AnimationComponent(size, new Animation.spriteList(sprites, stepTime: 0.01));
 ```
 
 If you have a sprite sheet, you can use the `sequenced` constructor, identical to the one provided by the `Animation` class (check more details in [the appropriate section](/doc/images.md#Animation)):
 
 ```dart
-    this.player = AnimationComponent.sequenced(64.0, 64.0, 'player.png', 2);
+    final size = Vector2(64.0, 64.0);
+    this.player = AnimationComponent.sequenced(size, 'player.png', 2);
 ```
 
 If you are not using `BaseGame`, don't forget this component needs to be update'd even if static, because the animation object needs to be ticked to move the frames.
 
-## SvgComponent
-
-This component uses an instance of `Svg` class to represent a Component that has a svg that is rendered on the game:
-
-```dart
-    Svg svg = Svg('android.svg');
-    SvgComponent android = SvgComponent.fromSvg(100, 100, svg);
-    android.x = 100;
-    android.y = 100;
-```
-
 ## FlareActor Component
```

Comment:
isn't Flare on a separated package now? cc @renancaraujo

Reply:
We keep all the docs here anyways right?